pycrostates/cluster/__init__.py

```diff
@@ -15,5 +15,6 @@
 to segment."""
 
 from .kmeans import ModKMeans  # noqa: F401
+from .aahc import AAHCluster
```

Comment:
```suggestion
from .aahc import AAHCluster  # noqa: F401
```

It's just to avoid warnings in IDEs ;)